lib/charms/nginx_ingress_integrator/v0/ingress.py

```diff
@@ -82,9 +82,7 @@
 OPTIONAL_CACHE_SETTING_RELATION_FIELDS = {
```

Comment:
What would be the impact of updating to the last published version (patch 10)?

Reply:
Tbh I have no idea, would definitely have to test that in later PRs.